### ISSUE TITLE
Ability to move folders

### ISSUE THREAD

**Issue by maxsptz** [loics2/sorta#11]
There should be a way to sort folders aswell!

**Comment by loics2** [comment]
Yeah, why not. This project is a kind of abandoned right now, but I have plenty of self-isolation time to update it!

**Comment by maxsptz** [comment]
any changes?
